lib/src/Dropdown/Dropdown.tsx

```diff
@@ -343,59 +384,51 @@ export const Dropdown: React.FC<DropdownProps> = (props: DropdownProps) => {
-                    name={props.name}
-                    value={props.selectedValue ? (props.selectedValue as DropdownItem).value : ""}
-                    onChange={props.onChange}
-                    id={props.id}
-                    placeholder={props.placeholder || null}
-                    multiple={!!props.multi}
-                >
-                    {props.list.map((item: DropdownItem) => (
-                        <option key={item.value} value={item.value}>
-                            {item.label}
-                        </option>
-                    ))}
-                </select>
-                {props.error && <div className="alert alert-danger custom-alert">{props.error}</div>}
-            </div>
-        );
-    }
-
     return (
-        <div className={`dropdown custom-dropdown${shouldDisable ? " disabled" : ""}${props.error ? " has-error" : ""}${props.className ? " " + props.className : ""}`} id={props.id}>
-            {props.label && <label className="dropdown-label">{props.label}</label>}
+        <div {...wrapperProps} className={classnames("dropdown custom-dropdown", { disabled: shouldDisable }, { "has-error": error }, wrapperProps?.className)}>
+            {label && <label className={classnames("dropdown-label", { disabled: shouldDisable })}>{label}</label>}
+
+            <select
+                {...props}
+                value={value || ""}
+                multiple={multiple}
+                disabled={shouldDisable}
+                className={classnames(`form-control custom-select custom-native-dropdown`, { disabled: shouldDisable }, props?.className)}
+                id={id}
+                hidden={!isMobile()}
+                onChange={(e) => onChange(multiple ? Array.from(e.target.selectedOptions, (option) => option.value) : e.target.value)}
+            >
+                {list.map((item: DropdownItem, i: number) => (
+                    <option hidden={!isMobile()} key={i} value={item.value}>
+                        {item.label}
+                    </option>
+                ))}
+            </select>
 
             <div
                 onKeyDown={shouldDisable ? null : handleKeyDownToggle}
                 ref={dropdownToggleRef}
-                className={`btn btn-secondary custom-dropdown-toggle${open ? " open" : ""}${props.more ? " more mx-right" : ""}${shouldDisable ? " disabled" : ""}`}
+                className={classnames(`btn btn-secondary custom-dropdown-toggle`, { open }, { more }, { "mx-right": more }, { disabled: shouldDisable }, props?.className)}
                 id={id}
                 aria-label={`Dropdown toggle: ${getTitleLabel()}`}
                 aria-haspopup={true}
                 aria-expanded={open}
                 tabIndex={shouldDisable ? -1 : 0}
                 onClick={shouldDisable ? null : handleClickToggle}
+                hidden={isMobile()}
```

Comment:
maybe we can make this not to render if it's in mobile? !isMobile() && <div ../>?